cmd/load-runner/main.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"regexp"
+	"time"
+)
+
+type config struct {
+	duration       string
+	parsedDuration time.Duration
+	namespace      string
+	loadName       string
+	chartFilesPath string
+}
+
+const (
+	defaultReleaseNamePrefix = "rudder-load"
+	defaultChartFilesPath    = "./artifacts/helm"
+)
+
+func main() {
```

Comment:
Could it be useful to do the usual context cancellation via signal? So that the program would catch the termination signal and do the proper cleanups and then exit?